test_aodntools/timeseries_products/test_aggregated_timeseries.py

```diff
@@ -60,6 +63,12 @@ def test_main_aggregator(self):
         self.assertIn(__version__, dataset.lineage)
         self.assertIn(BAD_FILE, dataset.rejected_files)
 
+        # check aggregated variable values
+        expected = Dataset(EXPECTED_OUTPUT_FILE)
+        non_match_vars = []
```

Comment:
I dont think you need this anymore

Reply:
I actually did mean to use this... updated.